papaya-core/src/main/java/pl/edu/agh/papaya/model/SprintStats.java

```diff
@@ -0,0 +1,30 @@
+package pl.edu.agh.papaya.model;
+
+import java.time.Duration;
+import javax.persistence.Embeddable;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@Embeddable
+public class SprintStats {
+
+    private Duration timeBurned;
+
+    private Duration estimatedTimePlanned;
+
+    private Duration finalTimePlanned;
+
+    private Double coefficient;
+
+    private Double averageCoefficientCache;
+
+    public void updateCoefficient(Duration totalDeclaredTime) {
+        if (timeBurned != null) {
+            this.coefficient = (double) totalDeclaredTime.toMinutes() / (double) timeBurned.toMinutes();
```

Comment:
Ten wzorek się zgadza z arkuszem?

Tam jest **chyba** coś takiego: `(((timePlanned - timeBurned) * totalDeclaredTime / timeBurned) + totalDeclaredTime) / finalTimePlanned`

Może z tego gdzieś wychodzi to samo, ale nie chce mi się liczyyć, więc poczekam, aż mi odpowiesz :stuck_out_tongue:

Reply:
Dobra, te komentarze na GitHubie żyją własnym życiem trochę jak dla mnie

https://github.com/papaya-agh/papaya-backend/pull/42#discussion_r285555901
>wychodzi na to, że estimatedTimePlanned nie jest koniecznie do liczenia czegokolwiek
to też nie będzie konieczny do podania przy updacie

Czy na pewno? Może mi się chrzanią te liczby, nie wiem

Reply:
sprawdzałem i te wzory są równoważne

Reply:
`(((finalTimePlanned - timeBurned) * totalDeclaredTime / timeBurned) + totalDeclaredTime) / finalTimePlanned = (((finalTimePlanned / timeBurned - 1) * totalDeclaredTime) + totalDeclaredTime) / finalTimePlanned = (((finalTimePlanned / timeBurned) * totalDeclaredTime)) / finalTimePlanned = (((1 / timeBurned) * totalDeclaredTime)) = totalDeclaredTime / timeBurned`

Reply:
EJ, RACJA